### DISCUSSION TITLE
Force certificate renewal from RSA to ECDSA

### DISCUSSION THREAD

**Discussion by CyberCr33p** [acmesh-official/acme.sh#4751] [General]
I try to switch from RSA to ECDSA for an already issued certificate using:

`acme.sh --renew --force --ecc -d example.com`

and I get:

[Mon Aug 21 13:36:50 EEST 2023] Renew: 'example.com'
[Mon Aug 21 13:36:50 EEST 2023] 'example.com' is not an issued domain, skip.

Possibly this happens because acme.sh doesn't find directory ~/.acme.sh/example.com_ecc and only ~/.acme.sh/example.com exists.

Without `--ecc` it renews it but with RSA.

Is it possible to "renew" it or the only way is to "issue" a new certificate?

**Comment by neilpang** [comment]
what is the cert type in the folder `~/.acme.sh/example.com` ?

If it was a RSA cert, it should only be renewd as RSA.

**Reply by CyberCr33p** [reply]
Yes it was a RSA cert.
